server/sonar-server/src/main/java/org/sonar/server/platform/ws/StatusAction.java

```diff
@@ -120,7 +124,24 @@ private boolean isConnectedToDB() {
     }
   }
 
-  private Status computeFromDbMigrationStatus() {
+  private Status computeStatusInStarting() {
+    DatabaseMigrationState.Status databaseMigrationStatus = migrationState.getStatus();
+    switch (databaseMigrationStatus) {
+      case NONE:
+        return Status.STARTING;
+      case RUNNING:
+        return Status.DB_MIGRATION_RUNNING;
+      case FAILED:
+        return Status.DOWN;
+      case SUCCEEDED:
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') This case's code block is the same as the block for the case on line 132. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1871)